Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM cyberbotics/webots 	
+
+RUN apt-get update && \ 	
+    apt-get install -y git rsync python3-yaml python3-jinja2 jq
```

Comment:
It seems you are not using jinja2 but raw string replace no?

Reply:
Same for `rsync` actually.

Reply:
> It seems you are not using jinja2 but raw string replace no?

True, I started with jinja2, but there was no need for it.